### ISSUE TITLE
selecting suggestion after typing [ should never result in an invalid selector

### ISSUE THREAD

**Issue by wackydude** [SelectorsHub/SelectorsHubIssues#9]
As you are building a selector in multiple parts, selector hub sometimes suggests possible selectors that are valid by themselves but result in an invalid selector when selected from the dropdown suggestions list.

Ex:
```
//button[@c
        @class="class1 class2"]
        [@class="class1 class2"]
```

Selecting the first suggestion results in the following invalid selector (note the missing [ between end of button and @ ):

`//button@class="class1 class2"]`

Possible solution:
Determine what selector type is being built, xpath or css, and only give suggestions that continue that same selector type. Alternatively, if an xpath is being built, and a css selector suggestion is selected, then the resulting selector should be a valid css selector.

![image](https://user-images.githubusercontent.com/28854812/91644842-cb47e500-e9f4-11ea-817f-e0b1d1099753.png)
![image](https://user-images.githubusercontent.com/28854812/91644845-d6027a00-e9f4-11ea-986b-8a9c474ff86d.png)

**Comment by selectorshubsanjay** [comment]
Thank you @wackydude for highlighting this issue. It is fixed and pushed with 1.0.4 release.
It is under review from Chrome side, very soon in 1-2days it will be available in store. Thank you for your support.

![image](https://user-images.githubusercontent.com/67390539/91650996-87acb400-eaa4-11ea-8b4a-564c152ade60.png)

![image](https://user-images.githubusercontent.com/67390539/91650998-92ffdf80-eaa4-11ea-995a-0971001df1b3.png)

**Comment by selectorshubsanjay** [comment]
Fix is live now. Please update the extension and test it.
Thank you.
